src/apis/index.js

```diff
@@ -138,8 +140,15 @@ export function postSkillFeedback(payload) {
   });
 }
 
+export function getUserIP() {
+  return ajax.get(`${USER_IP_URL}`, {
+    token: 'b3398b832bf0d7',
+  });
+}
+
 export function getDefaultMapData() {
-  return ajax.get('http://api.ipstack.com/134.201.250.155', {
-    access_key: '31fe38d0ec112087bc85f47241a58387',
+  const ip_user = getUserIP();
```

Comment:
This is not supposed to work. getUserIP() returns a promise and not an actual ip address. Did you test this code?